src/sentry/rules/processing/processor.py

```diff
@@ -302,6 +334,42 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
                 )
                 return
 
+        predicate_func = get_match_function(condition_match)
+        if not predicate_func:
```

Comment:
We should only error here if there is at least one slow or fast condition present - in the loop, we continued if `predicate_list` was empty.  `not predicate_func and (slow_conditions or fast_conditions)` (We can't use `condition_list` since it might not contain all conditions)